tests/test_preprocessors.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from corporacreator import preprocessors
+
+
+@pytest.mark.parametrize('locale, client_id, sentence, expected', [
```

Comment:
I'd suggest removing tests that no longer make sense in like of digits being banned. For example
```python
('fr', '*', "donc, ce sera 299 € + 99 €", "donc, ce sera deux cent quatre-vingt-dix-neuf euros plus quatre-vingt-dix-neuf euros"),
```
Some of the tests here, for example
```python
('fr', '*', "Jean-Paul II.", "Jean-Paul deux.")
```
have nothing to do with digits and can actually be run independent of this [comment](https://github.com/mozilla/CorporaCreator/pull/87#issuecomment-466296310).